app/src/main/java/net/tensory/rxjavatalk/providers/DebtProvider.java

```diff
@@ -16,7 +16,7 @@ public DebtProvider(DebtFeed debtFeed) {
         this.debtFeed = debtFeed;
     }
 
-    public Observable<Double> getLatestForHouse(House house) {
+    public Observable<Double> observeDebt(House house) {
```

Comment:
minor renaming, just to be clear that a hot observable is returned

Reply:
Ooh, good call.